src/js/components/Drop/DropContainer.js

```diff
@@ -81,8 +82,21 @@ const DropContainer = forwardRef(
             container.style.maxHeight = '';
           }
           // get bounds
-          const targetRect = findVisibleParent(target).getBoundingClientRect();
```

Comment:
I'm wondering why we even need `findVisibleParent()` here. I would think that we could just call `target.getBoundingClientRect()`. Is there a reason that won't work?

Reply:
`findVisibleParent()` is a recursive function that will look for the parent's element with a valid offsetParent property incase the element itself doesn't have offsetParent. 
SVG is one example of an element not having an offsetParent, but there can be more reasons for that that we haven't researched yet, so it is too soon to remove the function completely.
I agree that if we'll find a valid solution that will be generic for all elements without offsetParent, this recursive lookup will no longer be required.

Reply:
I just changed it to: `export const findVisibleParent = element => element;` and ran through the stories in storybook and ran the unit tests and didn't notice anything amiss. I'm not sure why we need this function.

Reply:
Do we have tests that check elements such as `li` or `svg`? You should see the difference using those as containers, and probably more elements that we need to research.

Reply:
If the DOM element hasn't loaded, I don't think we should show the Drop.

If other elements are different, we should add stories.

Reply:
Your suggestion made me realize how much of the complexity could be removed by not accounting for offsetParent. So I cleaned-up the PR -- Thank you. 
`findVisibleParent` is still used within the Layer component so I'll keep it for now, and if relevant, we could remove the usage of it there as well after verifying that it has the correct test coverage.